src/index.html

```diff
@@ -11,7 +11,184 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form name="form">
+      <fieldset class="perInfo" name="Personal">
+        <legend>Personal information</legend>
+        <label for="surName">
+          Surname:
+          <input 
+            type="text"
+            name="username" 
+            required
+          >
+        </label>
+
+        <label for="name">
+          Name:
+          <input 
+            id="name"
+            type="text"
+            name="name" 
+            required
+          >
+        </label>
+
+        <label for="age">
+          How old are You?
+          <input 
+            id="age" 
+            class="age"
+            type="number" 
+            name="age" 
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+
+        <label for="date">
+          Full date of birth:
+          <input 
+            id="date" 
+            type="date"
+            name="date" 
+            required
+          >
+        </label>
+
+        <label for="checkbox">
+          I accept the term of the agreement
+          <input 
+            id="checkbox" 
+            type="checkbox" 
+            name="agreement" 
+            required
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="regist">
```

Comment:
I would split same blocks which are on the same level
```suggestion
      </fieldset>
      
      <fieldset class="regist">
```